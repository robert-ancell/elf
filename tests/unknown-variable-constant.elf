unknown value = 42
print (value)
