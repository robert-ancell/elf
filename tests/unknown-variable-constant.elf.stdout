Line 1:
unknown value = 42
^^^^^^^
Unknown data type
